sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -67,7 +67,8 @@ import {
   SFASApplicationService,
   SFASPartTimeApplicationsService,
 } from "@sims/services/sfas";
-
+import { UserAESTController } from "./route-controllers/user/user.aest.controller";
```

Comment:
Please add the new controller to the controllers `index.ts` and import from there.